src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVIntervalTree.java

```diff
@@ -0,0 +1,904 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
```

Comment:
At the moment, the only use is as a sentinel (to mark the end of a stream of intervals).  But sure, your example is a good one.  I'll probably need them when I write the code that tries to piece together breakpoint evidence across partition boundaries, but maybe not.  I don't know.  Just seems useful and consistent to be able to shrink an interval to a point, I guess.